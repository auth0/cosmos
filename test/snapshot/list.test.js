import React from 'react'
import renderer from 'react-test-renderer'
import { List } from '@auth0/cosmos'
import 'jest-styled-components'

describe('List', () => {
  it('renders with automation attribute', () => {
    const tree = renderer
      .create(
        <List>
          <div>element</div>
        </List>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
