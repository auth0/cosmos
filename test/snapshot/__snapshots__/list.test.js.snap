// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders with automation attribute 1`] = `
.c0 {
  margin: 32px 0;
}

.c1 {
  border-top: 1px solid #DDD;
  padding: 16px 8px;
}

.c1:hover {
  background: #FAFAFA;
}

<div
  className="c0"
  data-cosmos-key="list"
>
  <div
    className="c1"
    data-cosmos-key="list.item"
  >
    <div>
      element
    </div>
  </div>
</div>
`;
